doghouse/server/cookieman/cookieman.go

```diff
@@ -0,0 +1,109 @@
+package cookieman
+
+import (
+	"encoding/base64"
+	"net/http"
+)
+
+type Cipher interface {
+	Encrypt(plaintext []byte) ([]byte, error)
+	Decrypt(ciphertext []byte) ([]byte, error)
+}
+
+type CookieMan struct {
+	defaultOpt CookieOption
+	cipher     Cipher
+}
+
+type CookieOption struct {
+	http.Cookie
+}
+
+type CookieStore struct {
+	name      string
+	cookieman *CookieMan
+	opt       *CookieOption
+}
+
+func (cs *CookieStore) Name() string {
+	return cs.name
+}
+
+func (cs *CookieStore) Set(w http.ResponseWriter, value []byte) error {
+	return cs.cookieman.Set(w, cs.name, value, cs.opt)
+}
+
+func (cs *CookieStore) Get(r *http.Request) ([]byte, error) {
+	return cs.cookieman.Get(r, cs.name)
+}
+
+func (cs *CookieStore) Clear(w http.ResponseWriter) {
+	cs.cookieman.Clear(w, cs.name)
+}
+
+func New(cipher Cipher, defaultOpt CookieOption) *CookieMan {
+	return &CookieMan{defaultOpt: defaultOpt, cipher: cipher}
+}
+
+func (c *CookieMan) NewCookieStore(name string, opt *CookieOption) *CookieStore {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported method CookieMan.NewCookieStore should have comment or be unexported